releasenotes/notes/upgrade-of-qiskit-nature-f808e73a446c1c93.yaml

```diff
@@ -0,0 +1,5 @@
+---
+upgrade:
+  - |
+    Upgrade the syntax to be compatible with the version 0.7 of the `qiskit-nature`.
```

Comment:
Can you add a few more lines to describe the changes? Also is there anything that directly impacts the users of `qiskit-cold-atom`? That might also be good to describe.

Reply:
So for the users I do not think that there is anything changing except that you should now use the syntax of `qiskit-nature` 0.7. @eggerdj maybe you could propose what we mainly changed there as those come mostly from you ?